frontends/web/src/routes/account/summary/coinbalance.tsx

```diff
@@ -70,21 +92,18 @@ export function CoinBalance ({
             </tr>
           </thead>
           <tbody>
-            { accounts.length > 0 ? (
-              coins.map(coinCode => {
-                if (accountsPerCoin[coinCode]?.length >= 1) {
-                  const account = accountsPerCoin[coinCode][0];
-                  return (
-                    <SubTotalCoinRow
-                      key={account.coinCode}
-                      coinCode={account.coinCode}
-                      coinName={account.coinName}
-                      balance={coinsBalances && coinsBalances[coinCode]}
-                    />
-                  );
-                }
-                return null;
-              })) : null}
+            { (Object.keys(coinsWithLightning).length > 0) ? (
+              Object.entries(coinsWithLightning).map(([coinCode, coinName]) => {
+                return (
+                  <SubTotalCoinRow
+                    key={coinCode}
+                    coinCode={coinCode as CoinCode}
+                    coinName={coinName}
+                    balance={coinsBalances && coinsBalances[coinCode]}
+                  />
+                );
+              }
+              )) : null}
```

Comment:
this looks incorrect:


<img width="1238" alt="Screenshot 2024-05-10 at 09 19 18" src="https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/dc98bff3-b408-4c6c-9050-10c0948a203f">

Reply:
it is correct, you have Bitcoin (in lightning) and also some bitcoin testnet coins.

but it could be sorted better as:
Bitcoin
Bitcoin Testnet
...